.bravo/config.yaml

```diff
@@ -12,28 +12,16 @@ pipeline:
         - |
           go mod download
           go build
-    
-    "Lint chart":
-      helm:
-        chartPath: charts/eventrouter
-        lint: true
 
     "Build Docker Image":
       imageBuild: &imageBuild
         push: false
         name: platform/eventrouter
         tags:
-          - 0.3.0.pes1.0
+          - 0.3.0.pes1.1
```

Comment:
Usually we only increase first number since we do not need more granular versioning here. `0.3.0.pes2`

Reply:
Done.